### PR TITLE
Branch add stall review and delete stall review

### PR DESCRIPTION
**Pull request by jamesebond**

### CODE REVIEW

File: src/main/java/seedu/address/model/stall/StallReview.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+import seedu.address.commons.util.AppUtil;
+
+/**
+ * Represents a review of a stall in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class StallReview {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Review should only contain alphanumeric characters and spaces, and it should not be BLANK!";
```
Comment (Ruizhi2001) [inline src/main/java/seedu/address/model/stall/StallReview.java:13]: will reviews be allowed to contain punctuations?

File: src/main/java/seedu/address/model/stall/StallReview.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    }
+
+    /**
+     * Returns true if a given string is a valid review.
+     *
+     * @param test The review string to validate.
+     * @return True if the review is valid, false otherwise.
+     */
+    public static boolean isValidReview(String test) {
+        return test.matches("[\\p{Alnum}][\\p{Alnum} ]*");
```
Comment (Ruizhi2001) [inline src/main/java/seedu/address/model/stall/StallReview.java:42]: if reviews are allowed to contain punctuations then this needs to be changed too

File: src/main/java/seedu/address/logic/commands/AddStallReviewCommand.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+ */
+public class AddStallReviewCommand extends Command {
+    public static final String COMMAND_WORD = "review-stall";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a stall review to the address book. "
+            + "Parameters: "
+            + "STALL INDEX (must be a positive integer) "
+            + PREFIX_RATING + "RATING "
+            + PREFIX_DESCRIPTION + "DESCRIPTION \n"
+            + "Example: " + COMMAND_WORD + " "
+            + "1 "
```
Comment (Ruizhi2001) [inline src/main/java/seedu/address/logic/commands/AddStallReviewCommand.java:27]: Do we need to include the store prefix here?

File: src/main/java/seedu/address/logic/commands/DeleteStallReviewCommand.java [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+ */
+public class DeleteStallReviewCommand extends Command {
+
+    public static final String COMMAND_WORD = "delete-stall-review";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes the stall review identified by the index number used in the displayed stall list.\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + ""
+            + PREFIX_STALL + "1";
```
Comment (samuelmui8) [inline src/main/java/seedu/address/logic/commands/DeleteStallReviewCommand.java:26]: `PREFIX_STALL` is included here but not in AddStallReviewCommand

File: src/main/java/seedu/address/model/stall/StallReview.java [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class StallReview {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Review should only contain alphanumeric characters and spaces, and it should not be BLANK!";
+    public static final String STAR_CONSTRAINTS =
+            "Rating should only be an integer between 0-5!";
+
+    private final String review;
+    private final int star;
```
Comment (samuelmui8) [inline src/main/java/seedu/address/model/stall/StallReview.java:18]: should these be abstracted out into classes to follow the AB3 OOP

File: src/main/java/seedu/address/model/stall/StallReview.java (hunk 2 above)
Comment (jamesebond) [reply src/main/java/seedu/address/model/stall/StallReview.java:42]: okAY MAKES SNESE

**Pull request closed without merging**